static/_app/tools/escaperoad/25011401/theme/game.css

```
@@ -0,0 +1,1291 @@
+
+body {
+	margin: 0;
+	background-color: #fafafa
+}
+
+@media print {
+	body {
+		background-color: #fff
+	}
+}
+.jss123 img{heigh:60px !important}
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)